### DISCUSSION TITLE
Add Age Verified Checkmarks to "Player List" and User Profiles in VRCX

### DISCUSSION THREAD

**Discussion by italiandogs** [vrcx-team/VRCX#941] [Ideas]
I developed a verification tool that uses one of the leading verification services, allowing Discord users—and by extension, VRChat users—to verify their age by submitting their ID. The service returns their date of birth (DOB) and records their username and DOB in a database, enabling Discord servers to automatically assign age-restricted roles without moderator intervention.

Proposal:
Integrating my age verification database into VRCX would provide users with real-time visibility into whether others in the current instance are age-verified (18+). By adding a simple checkmark next to verified users' profiles under the "Player List" and within individual user profiles, users could quickly identify who is verified and who is not, enhancing safety and transparency within the VRChat community.

**Comment by Natsumi-sama** [comment]
As much as I agree with children being exposed to things they shouldn't being a massive unresolved issue in VRChat, I don't believe age verification or VRCX to be the solution.

**Reply by Natsumi-sama** [reply]
Here is VRChats stance: https://ask.vrchat.com/t/developer-update-11-july-2024/25514#age-verification-10
Something like this needs to be a first party solution with how complicated and delicate it is handling other peoples private information, this would be a disaster waiting to happen in countless ways.

**Reply by italiandogs** [reply]
As much as I wish this was a non-issue, I have simply implemented the same optional verification system that Roblox currently has in place. With the more mature communities within VRC, this would give admins/moderators a way of making sure that anyone in the current instance is at least 18+ without releasing their age.

With a game community made up of all ages and communities that need to be restricted, age verification might be the only solution. Same with showing a bouncer at a bar or a nightclub your ID to verify they are drinking age. 

However, regardless of this solution, I am open to all ears on other possible ideas.
